src/containers/Login/index.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Button, Form, Input } from 'antd';
+import { LockOutlined, UserOutlined } from '@ant-design/icons';
+
+import styles from './style.module.css';
+
+const { Password } = Input;
+const { Item: ItemForm } = Form;
+
+const roleRequired = (message) => {
+  return{
+    required: true,
+    message,
+  }
+}
+
+const LoginContainer = ({ onSubmit }) => {
+  return (
+    <div className={styles.container}>
+        <h1>Estoque - RP</h1>
+        <img src="../../retina.png" alt="company logo" />
+        <Form name="signIn" onFinish={onSubmit} onFinishFailed={() => {}}>
```

Comment:
Se não iremos usar não precisa adicionar `onFinishFailed={() => {}}`

```suggestion
        <Form name="signIn" onFinish={onSubmit}>
```